.github/workflows/extract-api.yaml

```diff
@@ -0,0 +1,73 @@
+# imodel-transformer Extract API Build
+
+name: Extract API
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "**.md"
+      - docs/**
+      - .github/CODEOWNERS
+      - common/changes/**/*.json
+      - "**/CHANGELOG.json"
+      - "**/CHANGELOG.md"
+
+jobs:
+  extract-api:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+
+      - name: Configure git
+        run: |
+          git config --local user.email imodeljs-admin@users.noreply.github.com
+          git config --local user.name imodeljs-admin
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 7.27.0
+
+      - name: Use Node.js 18
```

Comment:
Node 18 EoL is coming up, maybe we should add an issue to update to 22

Reply:
good call. don't think this is the only occurrence. I'll check and maybe open a new pr just for the sake of easier tracking

Reply:
Yes,18 is used across the board it seems, should be a new PR. Just wanted to bring attention to it. Resolving